api_tests/nodes/views/test_node_draft_registration_list.py

```diff
@@ -166,7 +166,7 @@ def test_cannot_view_draft_list(
         project_public.remove_osf_group(group)
         project_public.add_osf_group(group, permissions.READ)
         res = app.get(url_draft_registrations, auth=group_mem.auth, expect_errors=True)
-        assert res.status_code == 403
+        assert res.status_code == 200
```

Comment:
I'm just assuming this is within the scope of the this ticket because OSF groups isn't exposed.